test/Streamly/Test/Data/Fold.hs

```diff
@@ -668,6 +668,18 @@ top = topBy True
 bottom :: Property
 bottom = topBy False
 
+nub :: Property
+nub = monadicIO $ action [1::Int, 1, 2, 3, 4, 4, 5, 1, 5, 7]
```

Comment:
There is no point in making `action` a function unless you pass both input and expected result to it.